tasks/generate/cache.rake

```diff
@@ -0,0 +1,65 @@
+namespace :generate do
+  desc "Generate a cache of lexers"
+  task :cache do
+    require "json"
+    require "rouge/util"
+    require "rouge/token"
+    require "rouge/lexer"
+    require "rouge/regex_lexer"
+    require "rouge/template_lexer"
```

Comment:
Why not just `require 'rouge'` instead?

Reply:
I'm not sure why I did it like this now and you're right to point it out. It should be `require 'rouge/all'` lest the generation of the cache be contaminated by the existing cache.

Reply:
Oh wait—I remember. It's wrong to bring in `rouge/all`. You're right to suggest `rouge`. The reason it didn't work like that was because when I wrote this code, `rouge` still imported all the lexers.